torchtune/datasets/_instruct.py

```diff
@@ -127,7 +127,7 @@ def instruct_dataset(
         tokenizer (Tokenizer): Tokenizer used to encode data. Tokenize must implement an `encode` and `decode` method.
         source (str): path string of dataset, anything supported by Hugging Face's `load_dataset`
             (https://huggingface.co/docs/datasets/en/package_reference/loading_methods#datasets.load_dataset.path)
-        template (str): class used to format the prompt. If the placeholder variable
+        template (str): full import path of class used to format the prompt. If the placeholder variable
```

Comment:
While you're here, can you add an example of using this from code and from YAML through the `Examples::` tag - similar to `chat_dataset`. This is helpful for anyone going through the documentation looking at the builder function.